src/database/models.py

```diff
@@ -519,6 +519,7 @@ class UserProfile(models.Model):
   created_at = models.DateTimeField(auto_now_add=True)
   updated_at = models.DateTimeField(auto_now=True)
   is_deleted = models.BooleanField(default=False, blank=True)
+  color = models.CharField(default="#000000", max_length = 7)
```

Comment:
I'm thinking that it would make sense to keep the color pref in a JSON field user_prefs.  We'll be adding more prefs for communication and viewing site stuff.